astroquery/query.py

```diff
@@ -463,3 +496,101 @@ def login(self, *args, **kwargs):
 
     def authenticated(self):
         return self._authenticated
+
+
+class RetrySession(Session):
+    """ Session that automatically does a number of retries for failed
+        transient errors. The time between retries double every time until a
+        maximum of 30sec is reached
```

Comment:
refer to the name of the variable that sets the 30s, instead of the value, in case it changes later